### PR TITLE
Added recipe for Flask-OpenID

### PR DESCRIPTION
**Pull request by pmlandwehr**

### CODE REVIEW

**Comment by pmlandwehr** [comment]
Seems like a bunch of `python3-openid` build crashed out before finishing. Restarted them, and will restart this once they are done.

**Comment by pmlandwehr** [comment]
Think these are fixed, so restarting CIs...

File: recipes/flask-openid/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+requirements:
+  build:
+    - python
+    - setuptools
+
+  run:
+    - python
+    - flask >=0.3  # [py2k]
+    - flask >=0.10.1  # [py3k]
```
Comment (pmlandwehr) [inline recipes/flask-openid/meta.yaml]: I have no idea, but that's what it says in [setup.py](https://github.com/mitsuhiko/flask-openid/blob/master/setup.py)

**Comment by jakirkham** [comment]
Thanks @pmlandwehr .

**Pull request merged**